.github/workflows/run_quic_interop.yml

```diff
@@ -10,7 +10,7 @@ jobs:
   run_quic_interop:
     strategy:
       matrix:
-        tests: [http3, transfer, handshake, retry, chacha20, resumption, amplificationlimit]
+        tests: [http3, transfer, handshake, retry, chacha20, resumption, amplificationlimit, ipv6]
```

Comment:
you probably also want to add this test to run_quic_interop_server.yml

Reply:
IMO for a separate PR.